app/Models/Feed.php

```diff
@@ -676,7 +676,7 @@ private function convertJSONFeedToRss(array $jf): ?string
 		$lastBuildDate = date(DATE_RSS, $latestDate);
 
 		//Create the RSS feed
-		$xmlFeed = new SimpleXMLElement('<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"></rss>');
```

Comment:
SimpleXMLElement cannot add elements with a CDATA node, which are necessary to preserve formatting tags in feed contents. the FreshRSS_sxml_Util class extends SimpleXMLElement with a method to add a CDATA child node